ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -1519,3 +1519,33 @@
                 timeout: 135
                 type: absolute
             publishers: []
+      - '{name}-{test_name}-for-pull-request':
+            test_name: kind-flexible-ipam-e2e
+            node: 'antrea-kind-flexible-ipam-testbed'
+            description: 'This is the {test_name} test for {name}.'
+            branches:
+            - ${{sha1}}      
+            builders:
+              - builder-kind-flexible-ipam-e2e:
+                  kind_cluster_name: '{test_name}'
```

Comment:
I passed this parameter to the builder so that it will create unique kind name `FULL_CLUSTER_NAME='{kind_cluster_name}'-"${{BUILD_NUMBER}}"` for each build.